src/saveload/saveload.h

```diff
@@ -352,6 +352,7 @@ enum SaveLoadVersion : uint16 {
 	SLV_MORE_CARGO_AGE,                     ///< 307  PR#10596 Track cargo age for a longer period.
 	SLV_LINKGRAPH_SECONDS,                  ///< 308  PR#10610 Store linkgraph update intervals in seconds instead of days.
 	SLV_AI_START_DATE,                      ///< 309  PR#10653 Removal of individual AI start dates and added a generic one.
+	SLV_COMPETITOR_CROSSINGS,               ///< 310  PR#todo  Added setting for disabling level crossings with competitors
```

Comment:
You don't need this. It's used to detect when the player loads a savegame created before this feature was added, so you can do things in `afterload.cpp` to fix their game. You don't do this, so you can remove this line entirely.

Reply:
Thanks, I've removed this change and the `from` line in world_setings.ini